### PR TITLE
Added parserlib jar for HeaderParser

### PR DESCRIPTION
**Pull request by araelamugithub**
Added Junit test cases for testing the header and the music files

Added sample abc music file

Added AbcNotations for testing the Header

### CODE REVIEW

File: src/abc/model/Music.java [hunk 1]
```diff
@@ -18,6 +18,17 @@ public class Music {
 	private int repeatStartTick = 0;
 	private boolean repeatSkip = false;
 
+	public static final String DEFAULT_METER = "4/4";
+    
+    public static final String DEFAULT_LENGTH = "1/8";
```
Comment (sanchewy) [inline src/abc/model/Music.java:23]: Nit: inconsistent indentation.

**Review by sanchewy — CHANGES_REQUESTED** [review]
Please do not copy paste this code. I can see the source here:
https://github.com/NarekDshkhunyan/abc-music-player/blob/438e84cff3b13404224df2cddd251fa060908c59/src/abc/parser/HeaderParser.java
Additionally the grammar is already defined, with header included. It doesn't really make sense to add another grammar that must be compiled just for the test cases. Reuse the existing grammar in this case.

File: test/abc/sound/MusicTests.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ * This contains the TestCases for testing the individual music files
+ */
+public class MusicTests {
+
+
+    @Test
+    public void testPlaySingleNote() throws InterruptedException {
+        String testFileName = "sample_abc/singlenote.abc";
+        Main.play(testFileName);  
+        Thread.sleep(1000);
```
[the remark this answers is not in the document]
Comment (araelamugithub) [reply test/abc/sound/MusicTests.java:17]: Yup, we do need it. If not, we will run into the different music playing at the same time

File: src/abc/model/Music.java (hunk 1 above)
Comment (araelamugithub) [reply src/abc/model/Music.java:23]: Will format it

**Pull request closed without merging**